src/components/_data/ContentSingularData.ts

```diff
@@ -2,5 +2,13 @@ export type ContentSingularData = {
     plainText_content?: string
     exampleImage_imageLink?: string
     exampleImage_percentageSize?: number
+    imagesInRow_items?: [
```

Comment:
Careful. What you have here means that it is now typed as an array with a single item in it. You should instead take the `ImgRowItem` type you defined earlier and do `ImgRowItem[]`. Alternatively it would be

```ts
{
    imgLink: string
    imgCaption: string
    imgAltTag: string
    externalLink: string
}[]
```